<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPlayer</class>
 <widget class="QMainWindow" name="QPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>344</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>344</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QT音乐</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SuourceFile.qrc">
    <normaloff>:/ICON/style/logo.png</normaloff>:/ICON/style/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>251</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="PrevButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="SuourceFile.qrc">
             <normaloff>:/Button/style/prev.png</normaloff>:/Button/style/prev.png</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="PlayButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="SuourceFile.qrc">
             <normaloff>:/Button/style/play.png</normaloff>:/Button/style/play.png</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="StopButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="SuourceFile.qrc">
             <normaloff>:/Button/style/stop.png</normaloff>:/Button/style/stop.png</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="NextButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="SuourceFile.qrc">
             <normaloff>:/Button/style/next.png</normaloff>:/Button/style/next.png</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="PauseButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="SuourceFile.qrc">
             <normaloff>:/Button/style/push.png</normaloff>:/Button/style/push.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="Phonon::VolumeSlider" name="volumeSlider" native="true"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Phonon::SeekSlider" name="seekSlider" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="MuButton">
       <property name="text">
        <string>目录</string>
       </property>
       <property name="icon">
        <iconset resource="SuourceFile.qrc">
         <normaloff>:/Memu/style/mu.png</normaloff>:/Memu/style/mu.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="AddButton">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset resource="SuourceFile.qrc">
         <normaloff>:/Memu/style/add.png</normaloff>:/Memu/style/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="RemoveButton">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset resource="SuourceFile.qrc">
         <normaloff>:/Memu/style/remove.png</normaloff>:/Memu/style/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="StyleButton">
       <property name="text">
        <string>随机</string>
       </property>
       <property name="icon">
        <iconset resource="SuourceFile.qrc">
         <normaloff>:/Memu/style/style.png</normaloff>:/Memu/style/style.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪雪峰体简</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>111</y>
      <width>250</width>
      <height>211</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="autoScrollMargin">
     <number>4</number>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DotLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="SuourceFile.qrc">
      <normaloff>:/ICON/style/logo.png</normaloff>:/ICON/style/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>56</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SuourceFile.qrc"/>
 </resources>
 <connections/>
</ui>
